_includes/partials/nextEvent.html

```diff
@@ -1,7 +1,7 @@
 <section class="module event-detail">
   <h2 class="module-heading">{{{ iconmonstr 'calendar-tick' }}}Next Event</h2>
   {{#each soonest}}
-    {{> eventDetail }}
+    {{> eventDetail site=../site }}
```

Comment:
@JakeSidSmith this was the issue!

the nested partial in `{{> nextEvent}}` renders `{{> eventDetail}}` scoped to this `{{#each soonest}}` loop context, so could no longer read from the global scope (where `site` lives)

fixed by hoisting a reference to `site` here `site=../site`

relative paths to read variables just reminds me how far we've progressed in this post-JSX era 😂

Reply:
Loops don't receive the outer context/scope? 🤨 That doesn't seem right.

<img width="789" alt="Screenshot 2024-03-14 at 16 37 55" src="https://github.com/asyncjs/async-website/assets/5850625/ae4db132-0767-4236-b02b-afc92a1a8fd0">


I'm happy to accept this as the fix though.